File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.py
def solution(a, b):
    answer = 0
    temp = 0
    
    if a == b:
        return a
    elif a > b:
        temp = a
        a = b
        b = temp
    
    for i in range(a, b+1):
        answer += i
    
    return answer

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.js
function solution(a, b) {
    var answer = 0;
    var temp = 0;
    
    if (a === b) {
        return a;
    } else if (a > b) {
        temp = a;
        a = b;
        b = temp;
    }
    
    for (i = a; i <= b; i++) {
        answer += i;
    }
    
    return answer;
}